show/main.py

```diff
@@ -2806,5 +2806,41 @@ def features():
         body.append([key, status_data[key]['status']])
     click.echo(tabulate(body, header))
 
+#
+# 'container' group
+#
+@cli.group(cls=AliasedGroup, default_if_no_args=False)
+def container()
+    """Show container"""
+    pass
+#
+# 'feature' group
+#
+@container.group(cls=AliasedGroup, default_if_no_args=False)
+def feature()
+    """Show container feature"""
+    pass
+
+# show whether the auto-restart feature for containers or a specific container
+# is enabled or disabled.
+
+@feature.command('autorestart')
+@click.argument('container_name', required=False)
+def autorestart(container_name):
+    """Show container feature autorestart"""
```

Comment:
This docstring gets used as the help statement for the command. Please make it a brief explanation of what the command does.

Reply:
Added an explanation.